test/support/style_case.ex

```diff
@@ -58,14 +57,13 @@ defmodule Styler.StyleCase do
   def style(code, style) do
     {ast, comments} = Styler.string_to_quoted_with_comments(code)
 
-    styled_ast =
+    {zipper, %{comments: comments}} =
```

Comment:
i'd like to unify the behavior implemented in style_case and task/style at some point (glueing together string_to_quoted_with_comments, a zipper traversal of a style, and going back to quoted_to_string)